src/App.vue

```diff
@@ -12,6 +12,9 @@
 import Navbar from "./components/Navbar";
 
 export default {
-    components: {Navbar}
+    components: {Navbar},
+    created() {
+        this.$router.push("/overview")
```

Comment:
Instead of doing `router.push` on app lifecycle created event you can create route with `/` Path which will redirect you to i.e. `overwiev`